library/core/src/ptr/mod.rs

```diff
@@ -882,9 +882,11 @@ pub const unsafe fn read_unaligned<T>(src: *const T) -> T {
 /// ```
 #[inline]
 #[stable(feature = "rust1", since = "1.0.0")]
-pub unsafe fn write<T>(dst: *mut T, src: T) {
+#[rustc_const_unstable(feature = "const_ptr_write", issue = "none")]
+pub const unsafe fn write<T>(dst: *mut T, mut src: T) {
     // We are calling the intrinsics directly to avoid function calls in the generated code
     // as `intrinsics::copy_nonoverlapping` is a wrapper function.
+    #[rustc_const_unstable(feature = "const_intrinsic_copy", issue = "80697")]
```

Comment:
Why is this attribute at the `extern` block, shouldn't it be at the `fn`?

Reply:
Oh, that explains it. Thanks :)